### DISCUSSION TITLE
Home Assistant X SenseCAP Indicator | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#637] [Q&A]
# Home Assistant X SenseCAP Indicator | Seeed Studio Wiki

SenseCAP Indicator x Home Assistant Application Development

https://wiki.seeedstudio.com/SenseCAP_Indicator_Application_Home_Assistant/

**Comment by pljda** [comment]
Hi and thank you for your demo and walkthrough.
if it is clear on "HA" chapter I get lost on your "Project Configuration" chapter. are you within Home Assistant or ??? 
in what file do you add the step 1 and 2?
thanks in advance
Philippe
